dissect/target/plugins/apps/browser/chromium.py

```diff
@@ -371,6 +378,119 @@ def extensions(self, browser_name: Optional[str] = None) -> Iterator[BrowserExte
                 except (AttributeError, KeyError) as e:
                     self.target.log.info("No browser extensions found in: %s", json_file, exc_info=e)
 
+    def _get_local_state_key(self, local_state_path: TargetPath, chromium_passwords: list) -> Union[bytes, bool]:
+        """Get the Chromium os_crypt encrypted_key and decrypt the key using DPAPI."""
+
+        if not local_state_path.exists():
+            self.target.log.warning(f"File {local_state_path} does not exist.")
+            return False
+
+        try:
+            local_state_conf = json.loads(local_state_path.read_text())
+        except json.JSONDecodeError:
+            self.target.log.warning(f"File {local_state_path} does not contain valid JSON.")
+            return False
+
+        if "os_crypt" not in local_state_conf:
+            self.target.log.warning(
+                f"File {local_state_path} does not contain os_crypt, Chrome is likely older than v80."
+            )
+            return False
+
+        encrypted_key = base64.b64decode(local_state_conf["os_crypt"]["encrypted_key"])[5:]
+        decrypted_key = self.target.dpapi.decrypt_blob(encrypted_key, chromium_passwords)
+        return decrypted_key
+
+    def passwords(self, browser_name: str = None, chromium_passwords: list = None) -> Iterator[BrowserPasswordRecord]:
+        """Return browser password records from Chromium browsers.
+
+        Chromium on Linux has ``basic``, ``gnome`` and ``kwallet`` methods for password storage:
+            - ``basic`` ciphertext prefixed with ``v10`` and encrypted with hard coded parameters.
+            - ``gnome`` and ``kwallet`` ciphertext prefixed with ``v11`` which is not implemented (yet).
+
+        Chromium on Windows uses DPAPI user encryption. Passwords from Chromium before and after version 80 can be
+        decrypted. The SHA1 hash of the user's password or the plaintext password is required to decrypt passwords.
```

Comment:
> before and after version 80 can be decrypted

I'm not up-to-date on my Chromium password lore, but without context this seems like a silly sentence? Did something happen in version 80 that warrants this statement? Maybe explain it by adding a sentence.